exporter/prometheusexporter/accumulator_test.go

```diff
@@ -444,8 +447,9 @@ func TestAccumulateDeltaToCumulative(t *testing.T) {
 			})
 			require.Equal(t, mLabels.Len(), vLabels.Len())
 			require.Equal(t, mValue, vValue)
+			require.Equal(t, dataPointValue1+dataPointValue2, vValue)
```

Comment:
Adding this assertion would fail the bug in comparison between the first and second data points timestamp.